platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -24,7 +24,7 @@ NS_ASSUME_NONNULL_BEGIN
  alternatively observe KVO change notifications to the pack’s `progress` key
  path.
  */
-extern NSString * const MGLOfflinePackProgressChangedNotification;
+extern NSNotificationName const MGLOfflinePackProgressChangedNotification;
```

Comment:
I stealthily edited [our versioning policy](https://github.com/mapbox/mapbox-gl-native/wiki/Versions-and-tagging#mapbox-ios-sdk) to carve out an exemption for Swift name refinement. The idea is that, with each major Swift revision, developers on the latest Swift will have to put up with a short period of API instability as we get our act together. It’s sort of like how you can upgrade to the latest iOS release the day it comes out, but application updates trickle in for weeks after that. Maybe that’s wrongheaded, but I see no other way we can keep up with MapKit, which has a latest-Swift-compatible release available the day the next version of Swift is officially released.

We can definitely target these changes to only affect Swift 3 code, by the way.